docs/source/recipes/torch-dataset-examples/utils.py

```
@@ -0,0 +1,192 @@
+def create_dataloaders_ddp(
+    dataset, get_item, cache_fields=None, local_process_group=None, **kwargs
+):
+    split_tags = ["train", "validation", "test"]
+    dataloaders = {}
+    for split_tag in split_tags:
+        split = dataset.match_tags(split_tag).to_torch(
+            get_item,
+            cache_fields=cache_fields,
+            local_process_group=local_process_group,
+        )
+        shuffle = True if split_tag == "train" else False
```

Ruff (0.8.2)

Remove unnecessary `True if ... else False`

Remove unnecessary `True if ... else False`

(SIM210)